frontend/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "frontend",
   "version": "0.1.0",
   "private": true,
+  "engines": {
+    "node": ">=20.0.0"
+  },
```

Comment:
If you were working locally with 20.8.0 we could also pin there? But it probably doesn't matter given it's just a minor version difference

Reply:
we're using `node:20-bullseye-slim` in the dockerfile, so this puts us in the same ballpark.